File: 2-Anno/Linguaggi/Java/Pratica/Simulazioni/N40/Auto.java
package N40;

public class Auto extends Veicolo {

	private int cilindrata;
	private double capienzaBagagliaio;

	public Auto(int codice, String tipo, String targa, String modello, String marca, double costoGiornaliero, int cilindrata, double capienzaBagagliaio) {
		super(codice, tipo, targa, modello, marca, costoGiornaliero);
		this.cilindrata = cilindrata;
		this.capienzaBagagliaio = capienzaBagagliaio;
	}

	public String toString() {
		return super.toString() + "\t" + cilindrata + "\t" + capienzaBagagliaio + "\t-\t-\t-\t";
	}
}

File: 2-Anno/Linguaggi/Python/Pratica/Simulazioni/N40/Auto.py
from Veicolo import Veicolo

class Auto(Veicolo):

	def __init__(self, codice, tipo, targa, modello, marca, costoGiornaliero, cilindrata, capienzaBagagliaio) -> None:
		super().__init__(codice, tipo, targa, modello, marca, costoGiornaliero)
		self._cilindrata = cilindrata
		self._capienzaBagagliaio = capienzaBagagliaio

	def __str__(self) -> str:
		return super().__str__() + "\t" + str(self._cilindrata) + "\t" + str(self._capienzaBagagliaio) + "\t-\t-\t-"
